changelog/unreleased/update-reva-beta8.md

```diff
@@ -1,7 +1,32 @@
-Enhancement: Update Reva
+Enhancement: Update Reva to version 2.10.0
 
-Update reva to latest edge.
+Changelog for reva 2.10.0 (2022-09-09)
+=======================================
+
+* Bugfix [cs3org/reva#3210](https://github.com/cs3org/reva/pull/3210): Jsoncs3 mtime fix
+* Enhancement [cs3org/reva#3213](https://github.com/cs3org/reva/pull/3213): Allow for dumping the public shares from the cs3 publicshare manager
+* Enhancement [cs3org/reva#3199](https://github.com/cs3org/reva/pull/3199): Add support for cs3 storage backends to the json publicshare manager
+
+Changelog for reva 2.9.0 (2022-09-08)
+=======================================
+
+* Bugfix [cs3org/reva#3206](https://github.com/cs3org/reva/pull/3206): Add spaceid when listing share jail mount points
+* Bugfix [cs3org/reva#3194](https://github.com/cs3org/reva/pull/3194): Adds the rootinfo to storage spaces
+* Bugfix [cs3org/reva#3201](https://github.com/cs3org/reva/pull/3201): Fix shareid on PROPFIND
+* Bugfix [cs3org/reva#3176](https://github.com/cs3org/reva/pull/3176): Forbid duplicate shares
+* Bugfix [cs3org/reva#3208](https://github.com/cs3org/reva/pull/3208): Prevent panic in time conversion
+* Bugfix [cs3org/reva#3881](https://github.com/cs3org/reva/pull/3881): Align ocs status code for permission error on publiclink update
+* Enhancement [cs3org/reva#3193](https://github.com/cs3org/reva/pull/3193): Add shareid to PROPFIND
+* Enhancement [cs3org/reva#3180](https://github.com/cs3org/reva/pull/3180): Add canDeleteAllHomeSpaces permission
+* Enhancement [cs3org/reva#3203](https://github.com/cs3org/reva/pull/3203): Added "delete-all-spaces" permission
+* Enhancement [cs3org/reva#4322](https://github.com/cs3org/reva/pull/4322): OCS get share now also handle received shares
```

Comment:
That's an ocis issue, not a reva PR. https://github.com/owncloud/ocis/issues/4322

Reply:
ok, reva PR was `3200`